### PR TITLE
Addition of reportSuspiciousActivitySettings

### PR DESCRIPTION
**Pull request by gregkmsft**
This docs PR is tracked via this work item: https://dev.azure.com/msft-skilling/Content/_workitems/edit/91700

### CODE REVIEW

File: api-reference/beta/resources/reportsuspiciousactivitysettings.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+doc_type: resourcePageType
+---
+
+# reportSuspiciousActivitySettings resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Defines the report suspicious activity settings for the tenant, whether it is enabled and which group of users is enabled for use.  Report suspicious activity enables users to report a suspicious voice or phone app notification multi-factor authentication prompt as suspicious.  These users will have their user risk set to high, and a risk event "User reported suspicious activity" is emitted.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/reportsuspiciousactivitysettings.md:16]: The last sentence ... is there a linkage to either the [riskDetection](https://learn.microsoft.com/en-us/graph/api/resources/riskdetection?view=graph-rest-beta) or [riskyUser](https://learn.microsoft.com/en-us/graph/api/resources/riskyuser?view=graph-rest-beta) APIs, specifically the riskEventType and riskDetail objects? Any new values being introduced? It's good to link back and call out any such new entries.

File: api-reference/beta/resources/reportsuspiciousactivitysettings.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|includeTarget|[includeTarget](../resources/includetarget.md)|Group IDs in scope for report suspicious activity.|
+|state|advancedConfigState|Specifies the state of the reportSuspiciousActivitySettings object. The possible values are: `default`, `enabled`, `disabled`, `unknownFutureValue`.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/reportsuspiciousactivitysettings.md:22]: Please indicate what the default state means.

File: api-reference/beta/resources/reportsuspiciousactivitysettings.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+|voiceReportingCode|Int32|Specifics the number the user will enter on their phone to report the MFA prompt as suspicious.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/reportsuspiciousactivitysettings.md:23]:
```suggestion
|voiceReportingCode|Int32|Specifies the number the user will enter on their phone to report the MFA prompt as suspicious.|
```

File: api-reference/beta/api/authenticationmethodspolicy-update.md [hunk 4]
```diff
@@ -48,6 +48,7 @@ In the request body, supply a JSON representation of the [registrationEnforcemen
 |:---|:---|:---|
 |registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|Enforce registration at sign-in time. This property can be used to prompt users to set up targeted authentication methods.|
 |systemCredentialPreferences|[systemCredentialPreferences](../resources/systemcredentialpreferences.md)|Prompt users with their most-preferred credential for multifactor authentication.|
+|reportSuspiciousActivitySettings|[reportSuspiciousActivitySettings](../resources/reportsuspiciousactivitysettings.md)|Enable users to report voice or phone app multi-factor authentication notifications as suspicious.|
```
Comment (FaithOmbongi) [inline api-reference/beta/api/authenticationmethodspolicy-update.md:51]: Maintain alphabetical order please.
```suggestion
|registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|Enforce registration at sign-in time. This property can be used to prompt users to set up targeted authentication methods.|
|reportSuspiciousActivitySettings|[reportSuspiciousActivitySettings](../resources/reportsuspiciousactivitysettings.md)|Enable users to report voice or phone app multi-factor authentication notifications as suspicious.|
|systemCredentialPreferences|[systemCredentialPreferences](../resources/systemcredentialpreferences.md)|Prompt users with their most-preferred credential for multifactor authentication.|
```

File: api-reference/beta/api/authenticationmethodspolicy-get.md [hunk 5]
```diff
@@ -123,6 +123,15 @@ Content-Type: application/json
...
     },
+    "reportSuspiciousActivitySettings": {
+        "@odata.type": "#microsoft.graph.reportSuspiciousActivitySettings",
+        "state": "enabled",
+        "includeTarget": {
+            "targetType": "group"
+            "id": "all_users",
+        },
+        "voiceReportingCode": 0,
+    },
```
Comment (FaithOmbongi) [inline api-reference/beta/api/authenticationmethodspolicy-get.md:134]: Reason for the failing validation.
```suggestion
    "reportSuspiciousActivitySettings": {
        "@odata.type": "#microsoft.graph.reportSuspiciousActivitySettings",
        "state": "enabled",
        "includeTarget": {
            "targetType": "group",
            "id": "all_users",
        },
        "voiceReportingCode": 0,
    },
```

File: api-reference/beta/api/authenticationmethodspolicy-update.md [hunk 6]
```diff
@@ -142,6 +151,14 @@ Content-Type: application/json
...
       ]
     }
+  },
+    "reportSuspiciousActivitySettings": {
+      "state": "enabled",
+      "includeTarget": {
+          "targetType": "group"
+          "id": "all_users",
+      },
+      "voiceReportingCode": 0,
```
Comment (FaithOmbongi) [inline api-reference/beta/api/authenticationmethodspolicy-update.md:161]:
```suggestion
  },
    "reportSuspiciousActivitySettings": {
      "state": "enabled",
      "includeTarget": {
          "targetType": "group",
          "id": "all_users",
      },
      "voiceReportingCode": 0,
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @gregkmsft for your work on this PR. I've left feedback to address inline.

**Review by jpettere — APPROVED** [review]

**Comment by jpettere** [comment]
No additional feedback from me. Faith caught everything I saw.

**Review by FaithOmbongi — APPROVED** [review]
Hi @gregkmsft - I've pushed a few commits relating to cross-linking to the riskDetection API docs and to fixing the API Doctor error then approved.

**Pull request merged**